Exec/gravity_tests/StarGrav/Prob_mhd_nd.F90

```diff
@@ -0,0 +1,140 @@
+
+! :::
+! ::: --------------------------------------------------------------------
+! :::
+subroutine ca_initmag(level, time, lo, hi, &
+                      nbx, mag_x, bx_lo, bx_hi, &
+                      nby, mag_y, by_lo, by_hi, &
+                      nbz, mag_z, bz_lo, bz_hi, &
+                      dx, xlo, xhi)
+
+  use probdata_module
+  use prob_params_module
+  use model_parser_module
+  use amrex_fort_module, only : rt => amrex_real
+
+  implicit none
+
+  integer :: level, nbx, nby, nbz
+  integer :: lo(3), hi(3)
+  integer :: bx_lo(3), bx_hi(3)
+  integer :: by_lo(3), by_hi(3)
+  integer :: bz_lo(3), bz_hi(3)
+  real(rt) :: xlo(3), xhi(3), time, dx(3)
+
+  real(rt) :: mag_x(bx_lo(1):bx_hi(1), bx_lo(2):bx_hi(2), bx_lo(3):bx_hi(3), nbx)
+  real(rt) :: mag_y(by_lo(1):by_hi(1), by_lo(2):by_hi(2), by_lo(3):by_hi(3), nby)
+  real(rt) :: mag_z(bz_lo(1):bz_hi(1), bz_lo(2):bz_hi(2), bz_lo(3):bz_hi(3), nbz)
+
+  real(rt) :: A_x(lo(1):hi(1), lo(2):hi(2)+1, lo(3):hi(3)+1)
+  real(rt) :: A_y(lo(1):hi(1)+1, lo(2):hi(2), lo(3):hi(3)+1)
+  real(rt) :: A_yin(lo(1):hi(1)+1, lo(2):hi(2), lo(3):hi(3)+1)
```

Comment:
is this used?

Reply:
good catch, I don't use A_yin, I used at some point for some experiments